Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -69,9 +69,17 @@ - (void)setUp
   _injectedObjects = [NSMutableDictionary new];
   [_socket setDelegateDispatchQueue:_jsQueue];
 
-  NSURL *startDevToolsURL = [NSURL URLWithString:@"/launch-js-devtools" relativeToURL:_url];
-  [NSURLConnection connectionWithRequest:[NSURLRequest requestWithURL:startDevToolsURL] delegate:nil];
-
+  NSURL *startDevToolsURL = [NSURL URLWithString:@"/launch-chrome-devtools" relativeToURL:_url];
+
+  //[NSURLConnection connectionWithRequest:[NSURLRequest requestWithURL:startDevToolsURL] delegate:nil];
+  NSURLRequest *request = [NSURLRequest requestWithURL:startDevToolsURL];
+  NSURLSession *session = [NSURLSession sharedSession];
+  NSURLSessionDataTask *dataTask = [session dataTaskWithRequest:request
```

Comment:
Can you remove the commented code, and just make this

`[session dataTaskWithRequest:[NSURLRequest requestWithURL:startDevToolsURL] completionHandler:NULL]`